src/components/App.js

```diff
@@ -1,7 +1,49 @@
-import React from 'react'
+import React, { useState } from 'react'
+import Button from 'components/Button/Button'
+import Container from 'components/Container/Container'
+import Input from 'components/Input/Input'
+import List from 'components/List/List'
+import ListItem from 'components/ListItem/ListItem'
 
 const App = () => {
-  return <></>
+  const [items, setItems] = useState([])
+  const [todo, setTodo] = useState('')
+
+  const handleKeyDown = (event) => {
+    if (event.keyCode === 13 && event.target.value !== '') {
+      setItems([...items, todo])
+      addTodo()
+    }
+  }
+
+  const addTodo = () => setTodo('')
+
+  const handleAddTodoButtonClick = () => {
+    setItems([...items, todo])
+    addTodo()
```

Comment:
Para corregirlo en referencia al comentario anterior:

```suggestion
    addTodo()
    setTodo('')
```